dotnet/samples/NCalcSkills/LanguageCalculatorSkill.cs

```diff
@@ -89,11 +89,15 @@ public LanguageCalculatorSkill(IKernel kernel)
         string input,
         SKContext context)
     {
-        var answer = await this._mathTranslator.InvokeAsync(input).ConfigureAwait(false);
+        SKContext answer;
 
-        if (answer.ErrorOccurred)
+        try
+        {
+            answer = await this._mathTranslator.InvokeAsync(input).ConfigureAwait(false);
```

Comment:
```suggestion
            var answer = await this._mathTranslator.InvokeAsync(input).ConfigureAwait(false);
```

Reply:
Related to the comment above - the `answer` variable is required in line #105.